java/ql/test/stubs/spring-ldap-2.3.2/org/springframework/ldap/support/LdapEncoder.java

```diff
@@ -0,0 +1,5 @@
+package org.springframework.ldap.support;
+
+public class LdapEncoder {
+  public static String filterEncode(String value) { return value; }
```

Comment:
This is a reason for the second false positive. The root cause is the same as in the previous case (`DefaultEncoder`).
Changing this method to return `null` fixes the problem.